src/h5web/visualizations/heatmap/IndexAxis.tsx

```diff
@@ -16,31 +16,33 @@ interface Props {
 
 function IndexAxis(props: Props): JSX.Element {
   const { orientation, numberPixels, className } = props;
+
   const [divRef, { width, height }] = useMeasure();
   const isLeftAxis = orientation === 'left';
+  const Axis = isLeftAxis ? AxisLeft : AxisBottom;
```

Comment:
This is the line that allows rendering one component or the other conditionally.